openmetadata-ui/src/main/resources/ui/src/components/Modals/EntityDeleteModal/EntityDeleteModal.tsx

```diff
@@ -45,6 +52,20 @@ const EntityDeleteModal = ({
   // To remove the entered text in the modal input after modal closed
   useEffect(() => {
     setName('');
+
+    // Using this method to autoFocus Input element since directly calling focus() doesn't work
+    // for the inputs inside modal. Ref - https://github.com/ant-design/ant-design/issues/8668
+    let timeout: number;
+
+    if (visible) {
+      timeout = window.setTimeout(() => {
+        deleteTextInputRef.current?.focus();
+      }, 1);
+    }
+
+    return () => {
+      clearTimeout(timeout);
```

Comment:
can we add check if timeout exists then only clear it out.

Reply:
Since clearTimeout accepts argument of type number or undefined, that will be handled by the function itself internally.